l10n_it_fiscalcode/README.rst

```diff
@@ -1,96 +1,65 @@
-    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
-    :alt: License: AGPL-3
-.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fl10n--italy-lightgray.png?logo=github
-    :target: https://github.com/OCA/l10n-italy/tree/12.0/l10n_it_fiscalcode
-    :alt: OCA/l10n-italy
-.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
-    :target: https://translation.odoo-community.org/projects/l10n-italy-12-0/l10n-italy-12-0-l10n_it_fiscalcode
-    :alt: Translate me on Weblate
-.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
-    :target: https://runbot.odoo-community.org/runbot/122/12.0
-    :alt: Try me on Runbot
-
-|badge1| |badge2| |badge3| |badge4| |badge5| 
-
-This module extends the functionality of partner to fit italian laws allowing you to compute Fiscal code for partners
-
-**Table of contents**
-
-.. contents::
-   :local:
-
-Installation
-============
-
-Install python package `codicefiscale <https://pypi.python.org/pypi/codicefiscale>`_.
-
-Usage
-=====
-
-Go to Partner and Run Wizard "Compute F.C."
-
-Bug Tracker
-===========
-
-Bugs are tracked on `GitHub Issues <https://github.com/OCA/l10n-italy/issues>`_.
-In case of trouble, please check there if your issue has already been reported.
-If you spotted it first, help us smashing it by providing a detailed and welcomed
-`feedback <https://github.com/OCA/l10n-italy/issues/new?body=module:%20l10n_it_fiscalcode%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
-
-Do not contact contributors directly about support or help with technical issues.
-
-Credits
-=======
-
-Authors
-~~~~~~~
-
-* Link IT s.r.l.
-* Apulia Software
-* Odoo Italia Network
-
-Contributors
-~~~~~~~~~~~~
-
-* Davide Corio
-* Luca Subiaco <subluca@gmail.com>
-* Simone Orsi <simone.orsi@domsense.com>
-* Mario Riva <mario.riva@agilebg.com>
-* Mauro Soligo <mauro.soligo@katodo.com>
-* Giovanni Barzan <giovanni.barzan@gmail.com>
-* Lorenzo Battistini <lorenzo.battistini@agilebg.com>
-* Roberto Onnis <onnis.roberto@gmail.com>
-* Franco Tampieri
-* Andrea Cometa <info@andreacometa.it>
-* Andrea Gallina
-* Matteo Bilotta <mbilotta@linkgroup.it>
-
-Maintainers
-~~~~~~~~~~~
-
-This module is maintained by the OCA.
-
-.. image:: https://odoo-community.org/logo.png
-   :alt: Odoo Community Association
-   :target: https://odoo-community.org
-
-OCA, or the Odoo Community Association, is a nonprofit organization whose
-mission is to support the collaborative development of Odoo features and
-promote its widespread use.
-
-This module is part of the `OCA/l10n-italy <https://github.com/OCA/l10n-italy/tree/12.0/l10n_it_fiscalcode>`_ project on GitHub.
-
-You are welcome to contribute. To learn how please visit https://odoo-community.org/page/Contribute.
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg

```

Comment:
Do not replace the dynamic generated README
See https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Reply:
I still see file replaced